.github/workflows/tests.yml

```diff
@@ -112,26 +112,7 @@ jobs:
         run: inv dev.cc faabric --shared
       - name: "Install Faabric shared library"
         run: inv dev.install faabric --shared
-      - name: "Build MPI native library"
-        run: inv dev.cc faabricmpi_native --shared
-      - name: "Install MPI native library"
-        run: inv dev.install faabricmpi_native --shared
       - name: "Build examples"
         run: inv examples
       - name: "Run example to check"
         run: inv examples.execute check
-
-  mpi_native:
```

Comment:
The `mpi_native` tests will now run as part of the distributed tests. What also means that will be checked against each new commit in a PR (not just when bumping docker images).